SmurfTestingScripts/README.md

```diff
@@ -2,34 +2,54 @@
 Python script used for automated StateSmurf testing.
 
 # CompareScenarios
-Script runs test-scenarios based on scenario.json file and compares each run's State transitions using [SmurfEvaluator](https://github.com/Melky-Phoe/StateSmurf/tree/master/SmurfEvaluator).  
-Test is successful when SmurfEvaluator is successful. Script doesn't consider exit code of application run.
+This script runs test-scenarios based on a scenario json file and compares each run's State transitions using [SmurfEvaluator](https://github.com/Melky-Phoe/StateSmurf/tree/master/SmurfEvaluator).  
+Test is successful when SmurfEvaluator is successful. The script considers the exit code of the application run.
 
+### Sequence diagram
+
+```mermaid
```

Comment:
I cannot open it :(.

Reply:
mermaid was chosen because it renders on github